Contest_API.md

```diff
@@ -1058,6 +1056,26 @@ clients with `admin` or `analyst` role. The `reaction` attribute
 is available to clients with `public` role only when the contest is
 not frozen.
 
+#### POST submissions
+
+To add submissions one can use the `POST` method on the submissions endpoint.
+The `POST` must include a valid JSON object with the same attributes the submission
+endpoint returns with a `GET` request with the following exceptions:
+
+* Any provided `id`, `reaction` and `contest_time` attributes are ignored.
```

Comment:
We might also want to explicitly disallow these, see also the wording about `id` in https://ccs-specs.icpc.io/contest_api#http-methods
But just saying "should not be provided, and are ignored" is fine with me.